bitnami/kubernetes-event-exporter/README.md

```diff
@@ -0,0 +1,92 @@
+# kubernetes-event-exporter
+
+[kubernetes-event-exporter](https://github.com/opsgenie/kubernetes-event-exporter) allows exporting the often missed Kubernetes events to various outputs so that they can be used for observability or alerting purposes.
+
+## TL;DR
+
+```console
+$ helm repo add bitnami https://charts.bitnami.com/bitnami
+$ helm install my-release bitnami/kubernetes-event-exporter
+```
+
+## Introduction
+
+This chart bootstraps a [kubernetes-event-exporter](https://github.com/opsgenie/kubernetes-event-exporter) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+
+## Prerequisites
+
+- Kubernetes 1.12+
+- Helm 2.11+ or Helm 3.0-beta3+
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+$ helm repo add bitnami https://charts.bitnami.com/bitnami
+$ helm install my-release bitnami/kubernetes-event-exporter
+```
+
+These commands deploy kubernetes-event-exporter on the Kubernetes cluster in the default configuration. The [Parameters](##parameters) section lists the parameters that can be configured during installation.
+
+> **Tip**: List all releases using `helm list` or `helm ls --all-namespaces`
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```console
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Parameters
+
+The following table lists the configurable parameters of the Ghost chart and their default values.
+
+| Parameter                               | Description                                                                                        | Default                                                    |
```

Comment:
Could you please divide this table in several smaller ones? It would help readability. You can use one of our [other READMEs ](https://github.com/bitnami/charts/tree/master/bitnami/opencart#parameters) as reference. 

Also, there are several missing parameters that are already included in `values.yaml` like `global.xxx` or `nameOverride`

Reply:
I updated the params table and divided them into smaller sections. Is it okay like this?

Reply:
Looks great! It seems only the `image.pullSecrets` parameter is still missing on the README.

Reply:
I am currently in the mountains for a few days. I will fix it next monday 😃

Reply:
Done :)